packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -99,7 +99,7 @@ export interface GridOptions {
    */
   disableColumnSelector?: boolean;
   /**
-   * If `true`, density selection is disabled.
+   * If `true`, density selector is disabled.
```

Comment:
?
It doesn't match the label in the documentation.

```suggestion
   * If `true`, the density selector is disabled.
```